abci/types/types.pb.go

```diff
@@ -328,7 +328,7 @@ func _Request_OneofMarshaler(msg proto.Message, b *proto.Buffer) error {
 		}
 	case nil:
 	default:
-		return fmt.Errorf("Request.Value has unexpected type %T", x)
```

Comment:
This file is auto-generated! We should probably change the associated `.proto` file instead.